pkg/proctree/types.go

```diff
@@ -0,0 +1,114 @@
+package proctree
+
+import (
+	"sync"
+
+	"github.com/RoaringBitmap/roaring"
+
+	"github.com/aquasecurity/tracee/pkg/utils/types"
+)
+
+// ProcessTree is a struct which follow the state of processes during runtime in container contexts.
+// The process tree is updated through Tracee's events, and is designed to overcome problems that may arise because
+// of events consumption (like handling lost events).
+type ProcessTree struct {
+	containers         types.RWMap[string, *containerProcessTree]
```

Comment:
The `containers` member is currently not used.
The idea is to keep track on the process trees of each container.
This is not mandatory now, so we can remove it.